packages/element-templates-json-schema-shared/src/defs/base-properties.json

```diff
@@ -144,6 +169,18 @@
           }
         }
       },
+      "feel": {
+        "$id": "#/properties/property/feel",
+        "type": "string",
+        "default": null,
+        "title": "property feel",
```

Comment:
I'd not use titles anymore, since they don't provide further information (context: https://github.com/camunda/element-templates-json-schema/pull/45). Most editors handle titles just as additional descriptions.